### DISCUSSION TITLE
Uninstall oh-my-bash in Linux

### DISCUSSION THREAD

**Discussion by cmonty14** [ohmybash/oh-my-bash#613] [General]
Hello,
I want to uninstall _oh-my-bash_ and remove it completely from my Linux machine.

Can you please advise how to proceed?

THX

**Comment by akinomyoga** [comment]
See [README](https://github.com/ohmybash/oh-my-bash?tab=readme-ov-file#uninstalling-oh-my-bash). The user configuration files are backed up as `~/.bashrc.omb-*`, so you can optionally remove those backup files after examination.
